zone/entity.cpp

```diff
@@ -3294,6 +3294,30 @@ void EntityList::ClearAggro(Mob* targ)
 	}
 }
 
+//removes "targ" from all hate lists of mobs that are water only.
+void EntityList::ClearWaterAggro(Mob* targ)
+{
+	Client *c = nullptr;
+	if (targ->IsClient())
+		c = targ->CastToClient();
+	auto it = npc_list.begin();
+	while (it != npc_list.end()) {
```

Comment:
Crazy formatting happening here

Reply:
Duplicated the function above:

//removes "targ" from all hate lists, including feigned, in the zone
void EntityList::ClearAggro(Mob* targ)
{
	Client *c = nullptr;
	if (targ->IsClient())
		c = targ->CastToClient();
	auto it = npc_list.begin();
	while (it != npc_list.end()) {

Do you want me to fix both to have braces?

Reply:
I agree.  I hate implied braces.  Too much time working on a military project where "do as they did" was enforced.  Fixing now.

Reply:
I fixed the function I added and the one I cloned.  This entire file has a ton of single line ifs without braces.  I feared breaking something if I changed everything.